cmd/hactar/commands/status.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/session"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/url"
+	"github.com/mkideal/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+type StatusParams struct {
+	cli.Helper
+}
+
+func RunStatusCommand(ctx *cli.Context) error {
+	lotusClient, err := lotus.NewClient(nil, nil)
+	if err != nil {
+		fmt.Println("Unable to initialize lotus client")    // show to user
```

Comment:
detect* This sound like we are starting one and might be confusing to our dummy users